CharlieBackend.AdminPanel/CharlieBackend.AdminPanel.csproj

```diff
@@ -11,6 +11,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\CharlieBackend.Business\CharlieBackend.Business.csproj" />
```

Comment:
Why add this project reference?

Reply:
Thnks.
https://github.com/ita-social-projects/WhatBackend/pull/464/commits/fd5438cdb81f454f15dfe285770e85112fa9c8fb